BadHabbit/BadHabbit/UserStories/SecondScreen/SecondScreenModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  SecondScreenModel.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 24.05.23.
+//
+
+import Foundation
+
+struct SecondScreenModel {
+    var results: [Petition]
+    init(dictionary: [String: Any]) {
+        self.results = []
+        for petition in dictionary["results"] as? [[String: Any]] ?? []{
+            results.append(Petition(dictionary: petition))
+        }
+    }
+}
+struct Petition{
+    let title: String
+    init(dictionary: [String: Any]) {
+        title = dictionary["title"] as? String ?? ""
```

Comment:
This logic should be performed in parser, not in model

Reply:
Changed